ui/components/app/transaction-decoding/index.scss

```diff
@@ -0,0 +1,169 @@
+//styling for ui components
+@import './components/ui/copy-raw-data/index';
+
+//styling for decoding components
+@import './components/decoding/address/index';
+
+.tx-insight {
+  overflow-x: hidden;
+
+  &-loading {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100px;
+
+    .spinner {
+      width: 30px;
+    }
+  }
+
+  & > details {
+    margin: 20px 0;
+  }
+
+  & > details > summary {
+    &.tx-insight-title {
+      font-size: 14px;
+      position: relative;
+      margin: 12px 0 4px;
+      padding-left: 22px;
```

Comment:
`padding-inline-start` would be good here.

Reply:
1efaa13e6